CMakeLists.txt

```diff
@@ -48,6 +48,8 @@ include(pybind11)
 include(torch)
 include(k2)
 include(kaldifeat)
+include(kaldi_native_io)
+include(sentencepiece)
```

Comment:
I don't think we really need sentencepiece in decoding, we always generate tokens for the bpe model. I suggest to pass a argument like `--use-bpe`, if it is True then we can replace the "▁" with " " in the recognized texts.

Reply:
Yes, you are right. Fixed.